src/main/java/sirius/biz/protocol/TraceData.java

```diff
@@ -99,6 +100,24 @@
     @Transient
     private boolean silent;
 
+    /**
+     * Returns a default selection of {@link DateRange date ranges} for a time aggregation on {@link #CHANGED_AT}.
+     *
+     * @return an array of date ranges to create a time aggregation on the last changed date
+     * @see sirius.biz.web.MongoPageHelper#addTimeAggregation(Mapping, boolean, DateRange...)
```

Comment:
+1 for providing a reference - maybe also link to the equivalent ES version in ElasticPageHelper(?)